tests/scancode/test_cli.py

```diff
@@ -52,7 +52,6 @@
 from scancode.cli_test_utils import run_scan_click
 from scancode.cli_test_utils import run_scan_plain
 
-
```

Comment:
Why removing that blank line? have 2 empty lines after the import is our convention.

Reply:
I got it.